### ISSUE TITLE
Pull Request description text is cut off in generated release notes

### ISSUE THREAD

**Issue by KacperWeiss** [rfennell/AzurePipelines#1512]
### Azure DevOps Extensions

Generate Release Notes (Node Cross Platform)

### Platform

Azure DevOps Services

### Describe the bug

When we create our Release Notes using this part in template:

```
# Associated Pull Requests ({{pullRequests.length}})
{{#forEach pullRequests}}
{{#if isFirst}}Associated Pull Requests (only shown if  PR) {{/if}}
## **PR{{this.pullRequestId}}**  {{this.title}}
{{this.description}}
{{/forEach}}
```

Is not working as expected. Descriptions from PRs are cut off at random points - each time with different length of characters for each description (one 510 characters long, another 480, another 560), and there was never any special character that could potentially cause it to end at those random places.

### Repo Steps

1. In build pipeline use Generate Release Notes task.
2. Use a part of template that will add list of associated PRs containing their title and description to generate release notes.
3. Make sure PRs have descriptions more than 1k characters long.
4. See file created with the task - it should have it's descriptions cut off at point in every PR.

### Expected Behavior

Release notes should contain full PR's description

**Comment by rfennell** [comment]
I will have to try to rept.

I wonder if the API call only provides a truncated description as is the case for commits unless you expand them with an extra call?

**Comment by rfennell** [comment]
Reproduced the problem in a test harness. The issue is as expected that the PR description is being truncated in the payload returned by the Azure DevOps API.

I will try to find a way to get the full message

**Comment by rfennell** [comment]
Just checked and this problem was previously reported in this project's issue #915,

Unfortunately with that issue we found no solution as it is a limitation of the Microsoft provided API call `Get Pull Requests By Project`, and it appears they have not addressed it since that they have not fixed it.

However, I have now come up with a solution and that is to make an extra call to get a single PR, just to get a non-truncated description.

This will be released as 4.7.x ASAP
